Makefile

```diff
@@ -1,26 +1,13 @@
-# Copyright 2017 The Kubernetes Authors.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-PLATFORM       ?= linux/amd64
 REGISTRY_NAME  ?= index.docker.io/linode
 IMAGE_NAME     ?= linode-blockstorage-csi-driver
-REV            := $(shell git describe --long --tags --dirty)
+REV            := $(shell git describe --long --tags --dirty 2> /dev/null || echo "dev")
```

Comment:
Why the echo? Is this because of the shallow clones in `actions/checkout`? Or is it because you're running git-less alpine there, so you don't have access to git history?